src/components/LoginScreen/index.js

```diff
@@ -1,37 +1,65 @@
-import React from "react";
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import appSchema from '../../validations/app';
+import LanguageContainer from '../../components/LanguageContainer';
+import useForm from 'react-hook-form';
+import Validation from '../../components/Validation';
 import { withRouter } from "react-router";
-import { useTranslation } from "react-i18next";
+
 import styles from "./index.module.scss";
 
 const LoginScreen = props => {
   const { t } = useTranslation();
+  const { register, handleSubmit, errors, triggerValidation, setError } = useForm({
+    validationSchema: appSchema(t),
+    mode: 'onBlur',
+  });
+
+  useEffect(() => {
+    return () => errors && triggerValidation();
```

Comment:
I don't understand what you do here

Reply:
Returning function in useEffect works like componentWillUnmount, now useEffect is listening to "t" so function will run at the end of change, it wont validate on mount and will validate only after change of "t"